contracts/CrowdsaleTokenExt.sol

```diff
@@ -6,6 +6,8 @@
 
 pragma solidity ^0.4.8;
 
+import "oracles-zeppelin/contracts/token/ERC20.sol";
```

Comment:
@fvictorio you don't need to import it. Because you've already imported it from `import './StandardToken.sol';`

Reply:
Oh, I didn't know that imports worked that way. Fixed, thanks.